tests/robottelo/decorators/test_host.py

```diff
@@ -12,139 +14,197 @@
 
 
 class GetHostOsVersionTestCase(TestCase):
+    """Tests for _get_host_os_version version"""
+
     def setUp(self):
+        """Mocking ssh"""
         self._patcher = mock.patch(
             'robottelo.decorators.host.ssh.command')
         self._command = self._patcher.start()
 
     def tearDown(self):
+        """Stop mock created on setUp method"""
         self._patcher.stop()
 
     def assert_rhel_version(self, ssh_version, parsed_version):
+        """
```

Comment:
Start the docstring just in front of opening `"""`

Also add a blank line between the docstring description and the :param:s definitions.

This is what most of robottelo code follows.